README.md

```diff
@@ -1296,7 +1296,6 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [Back To Top](#contents) |
 
 - ### **P**
-
```

Comment:
There is nothing it's a blank line. Previously I added my name to it & now I changed this to normal. It's fine. It's not a problem.

Reply:
It means that there should be a line space between `- ### **P**` and `- [Pablo Lorenzo](https://github.com/Ll2NZ)`. Since you removed the line, GitHub conflict occurs meaning there are overlapping lines. Please see the original `README.md` here in the repo not in your local.